spire-flex/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
None of this content in NOTES.txt will be useful I think to the end result. Should we just remove the file for now?

Reply:
New notes looks great.

Reply:
Marking this item resolved